app/src/main/res/layout/activity_main.xml

```diff
@@ -1,37 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:fab="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools" android:id="@+id/activity_main"
-    android:layout_width="match_parent" android:layout_height="match_parent"
-    android:paddingBottom="@dimen/activity_vertical_margin"
-    android:paddingLeft="@dimen/activity_horizontal_margin"
-    android:paddingRight="@dimen/activity_horizontal_margin"
-    android:paddingTop="@dimen/activity_vertical_margin"
-    tools:context="com.doublesp.coherence.activities.MainActivity">
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
 
-    <android.support.v7.widget.RecyclerView android:id="@+id/recycler_view"
-        android:layout_width="match_parent" android:layout_height="match_parent" />
+    <RelativeLayout xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/activity_home" android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context="com.doublesp.coherence.activities.MainActivity">
 
-    <com.getbase.floatingactionbutton.FloatingActionsMenu android:id="@+id/multiple_actions"
```

Comment:
we don't need "recipe" button in custom list view, instead it will appear in "saved recipes" page.